Cargo.toml

```diff
@@ -12,6 +12,7 @@ readme = "README.md"
 [dependencies]
 winrt_macros = { path = "crates/macros",  version = "0.7" }
 sha1 = "0.6.0"
+wchar = "0.6.0"
```

Comment:
I'd like to avoid dependencies as much as possible. If we can void the `wchar` dependency that would be great.

Reply:
I generally want to prefer avoiding a dep for a small usage, but in this case the alternative do kind of suck too.
Which would you prefer: copying the `wch_c!` macro into your own macro crate, manually expanding the strings to u16 slices or dynamically converting the str literals to utf-16?

Reply:
Whatever is simplest. I'd rather not copy the macro, but just inline the conversion since its only needed in one place.

Reply:
Just to be clear, the point of wchar is that the conversion is static, e.g. it emits `&[65u16, ...]`, so do you mean change it to be dynamically converting?

Reply:
Yes, that would be ideal. If you can do that without too much effort that's great. Otherwise, since it is behind a `call_once` I don't think its the end of the world if the conversion is dynamic. None of the functions that we dynamically load are in hot code paths.